### ISSUE TITLE
Compressed WAD file support.

### ISSUE THREAD

**Issue by SteelT1** [sirjuddington/SLADE#699]
**SLADE Version**: (3.1.1.5)
**Operating System**: Windows

**Issue Details**:
The ability to read/edit compressed WAD would be nice. If you want, I can give you a source to base it off from.

**Comment by Gaerzi** [comment]
Please do, I'm not sure what compressed WAD file are exactly.

**Comment by SteelTitanium** [comment]
Here is an source to wadzip, which what allows you to make a compressed WAD file. http://git.magicalgirl.moe/STJr/wadzip/tree/master

**Comment by sirjuddington** [comment]
Do any ports support this format? If not I'm not sure there is much reason to add it

**Comment by Blzut3** [comment]
Although that link is dead, STjr in the URL suggests it's an SRB2 thing: https://wiki.srb2.org/wiki/WAD_file#ZWAD, https://github.com/STJr/SRB2/tree/master/tools/wadzip

Don't personally know how widespread the usage is, especially given SRB2 supports pk3 now.

**Comment by SteelT1** [comment]
Right, so when I originally made this issue. It was because back then SRB2 didn't have pk3 support so zwad was the only way people could compress files.

But pk3 has been supported by the game since v2.1.21, and I think pretty much most of the game's community use pk3 now for mods, so I don't know how widespread the usage of zwad is today still myself, that's worth adding support for it in SLADE at all.

**Comment by sirjuddington** [comment]
Ok, in that case I'll close this one, since as far as I know outside of this github issue, lack of zwad support hasn't ever been brought up (to me) since, so it's probably safe to leave it unsupported
